### PR TITLE
trying out emoji response for so-good and look-at-that

### PR DESCRIPTION
**Pull request by hitjim**
Added some gif emoji to the slack, and corresponding emoji response commands instead of bombing with image links

Resolves Issue #43

### CODE REVIEW

**Comment by hitjim** [comment]
Update PR to use hubot-slack-reaction

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "hubot-secret-ballot": "0.0.2",
     "hubot-shipit": "^0.1.1",
     "hubot-slack": "^3.3.0",
+    "hubot-slack-reaction": "0.0.3",
```
Comment (jcbwlkr) [inline package.json]: Drop this down 0.0.2 to avoid a bug introduced in 0.0.3 then we're golden

**Comment by jcbwlkr** [comment]
Okay actually it seems that `hubot-slack-reaction` version `0.0.3` is fine and the problem is that our `hubot-slack` is too old. Please update `hubot-slack` to `^4.2.1` then I'll merge and deploy.

**Comment by hitjim** [comment]
BLOOP

**Pull request merged**